tensorflow/core/util/gpu_launch_config.h

```diff
@@ -224,12 +224,12 @@ inline GpuLaunchConfig GetGpuLaunchConfigFixedBlockSize(
   //CHECK_EQ(err, hipSuccess);
 
```

Comment:
HIP doesn't support `hipOccupancyMaxActiveBlocksPerMultiprocessor` yet so we can't merge CUDA/ROCm paths here yet.